devolutions-gateway/src/http/controllers/jet.rs

```diff
@@ -64,31 +64,39 @@ impl JetController {
         };
 
         // check the session token is signed by our provider if unrestricted mode is not set
-        if !self.config.unrestricted {
-            match validate_session_token(self.config.as_ref(), &req) {
-                Err(e) => {
-                    slog_scope::error!("Couldn't validate session token: {}", e);
-
-                    return (StatusCode::UNAUTHORIZED, ());
-                }
-                Ok(expected_id) if expected_id != association_id => {
+        let jet_tp_claim = match validate_session_token(self.config.as_ref(), &req) {
```

Comment:
same comment here as in the other place where the jet_tp claim is extracted: does this correctly handle the case where the token is omitted? old versions of Wayk Bastion will sign tokens that don't contain this claim, and we need to handle it properly (no recording when the claim is absent).

Reply:
yes, the jet_tp claim has the type Option<>. So if the jet_tp claim will be absent, the value will be set to None and the recording will not start.